src/main.js

```diff
@@ -427,7 +427,7 @@ class MiniGraphCard extends LitElement {
   }
 
   renderSvgPoints(points, i) {
-    if (!points) return;
```

Comment:
I don't think, this PR is complete. If we remove this statement, the passed parameter is useless and should also be removed. But I reckon the `points` passed into here are used elsewhere, too...

So, we should either `calcPoints` everywhere they are used. Or pass them around and make sure they are defined in the central place.

I'll have to look at this more closely, when I find the time.

Reply:
`points` is still used when redering the actual points as it is mapped further down in this function.